static/src/javascripts/projects/common/modules/ui/subscription-banners/subscription-banner-tracking.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+import {
+    submitViewEvent,
+    submitClickEvent,
+    addTrackingCodesToUrl,
+} from 'common/modules/commercial/acquisitions-ophan';
+import config from 'lib/config';
+import { trackNonClickInteraction } from 'common/modules/analytics/google';
+
+// types
+import type { ReaderRevenueRegion } from 'common/modules/commercial/contributions-utilities';
+
+export type BannerTracking = {
+    signInUrl: string,
+    gaTracking: () => any,
+    subscriptionUrl: string,
+    trackBannerView: () => void,
+    trackBannerClick: (button: any) => void,
+    trackCloseButtons: (button: any) => void,
+};
+
+const DISPLAY_EVENT_KEY = 'subscription-banner : display';
```

Comment:
might want to consider pulling out the prefix `subscription-banner: ` since it is repeated a few times.